packages/renderer/src/renderer.ts

```diff
@@ -0,0 +1,142 @@
+import jsonld from '@digitalcredentials/jsonld';
+import { JsonLdObj } from 'jsonld/jsonld-spec';
+
+import {
+  VerifiableCredential,
+  RenderMethodPayload,
+  IRendererProvider,
+  IAgentPlugin,
+  IRenderCredentialArgs,
+  IRendererContext,
+  IRenderResult,
+  UnsignedCredential,
+  IRenderer,
+} from '@vckit/core-types';
+import schema from '@vckit/core-types/build/plugin.schema.json' assert { type: 'json' };
+
+export const RENDER_METHOD = 'https://www.w3.org/2018/credentials#renderMethod';
+
+/**
+ * Renderer class responsible for rendering verifiable credentials using specified render methods.
+ * @public
+ */
+export class Renderer implements IAgentPlugin {
```

Comment:
to keep the naming consistent with other plugins , i suggest to rename it to `RendererPlugin`